### PR TITLE
TAP HERE TO SIGN IN bug fixed

### PR DESCRIPTION
**Pull request by dev-raj-kumar**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -31,5 +31,7 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+
```
Comment (krharsh17) [inline app/build.gradle]: Remove extra newline

File: app/build.gradle [hunk 2]
```diff
@@ -31,5 +31,7 @@ dependencies {
...
     implementation 'com.journeyapps:zxing-android-embedded:3.4.0'
+
```
Comment (krharsh17) [inline app/build.gradle]: same

File: app/src/main/java/in/tinyhouse/salesforce/Onboarding/SignUpActivity.java [hunk 3]
```diff
@@ -13,6 +13,10 @@
 import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
 
+
+
+
+
```
Comment (krharsh17) [inline app/src/main/java/in/tinyhouse/salesforce/Onboarding/SignUpActivity.java]: same

File: app/src/main/java/in/tinyhouse/salesforce/Onboarding/SignUpActivity.java [hunk 4]
```diff
@@ -195,16 +199,20 @@ public boolean checkEntries(String name, String phone, String email, String pass
         }
         if ("".equals(email)) {
             Toast.makeText(SignUpActivity.this, "EMAIL cannot be empty", Toast.LENGTH_SHORT).show();
+
```
Comment (krharsh17) [inline app/src/main/java/in/tinyhouse/salesforce/Onboarding/SignUpActivity.java]: same

File: app/src/main/java/in/tinyhouse/salesforce/Onboarding/SignUpActivity.java [hunk 5]
```diff
@@ -195,16 +199,20 @@ public boolean checkEntries(String name, String phone, String email, String pass
...
             return false;
+
```
Comment (krharsh17) [inline app/src/main/java/in/tinyhouse/salesforce/Onboarding/SignUpActivity.java]: same

File: app/src/main/java/in/tinyhouse/salesforce/Onboarding/SignUpActivity.java [hunk 6]
```diff
@@ -195,16 +199,20 @@ public boolean checkEntries(String name, String phone, String email, String pass
...
             return false;
+        
         } else if ("".equals(password)) {
             Toast.makeText(SignUpActivity.this, "PASSWORD cannot be empty", Toast.LENGTH_SHORT).show();
             return false;
         } else
             return true;
     }
 
+
```
Comment (krharsh17) [inline app/src/main/java/in/tinyhouse/salesforce/Onboarding/SignUpActivity.java]: same

File: app/src/main/java/in/tinyhouse/salesforce/Onboarding/SignUpActivity.java [hunk 7]
```diff
@@ -195,16 +199,20 @@ public boolean checkEntries(String name, String phone, String email, String pass
...
     public void failedSignUpSnackbarMessage(String message) {
         Snackbar.make(findViewById(R.id.root_signuplayout), "Error " + message, Snackbar.LENGTH_SHORT).show();
     }
 
+
```
Comment (krharsh17) [inline app/src/main/java/in/tinyhouse/salesforce/Onboarding/SignUpActivity.java]: same

File: app/build.gradle (hunk 1 above)
Comment (dev-raj-kumar) [reply app/build.gradle]: extra newlines means sir??

Comment (dev-raj-kumar) [reply app/build.gradle]: @krharsh17

Comment (krharsh17) [reply app/build.gradle]: the empty line shown in green

**Pull request closed without merging**